### ISSUE TITLE
OTR: meaning of "quorum sensing involved in interaction with host"

### ISSUE THREAD

**Issue by krchristie** [geneontology/go-ontology#17757]
There may be a need for the term `quorum sensing involved in interaction with host`, but the existing annotations do not seem to reveal it.

- 2 annotations from Fujiya et al. 2007 (1 IMP to human SLC22A5 & 1 IDA to Bacillus subtilis phrC) - This paper characterizes the host response to a secreted factor from B. subtilis that happens to be a quorum sensing molecule; it does not talk about the role of phrC in quorum sensing. I think the mammalian protein could be annotated to the existing term 'response to bacterium' or perhaps 'response to symbiotic bacterium'. Or, if people think it's useful, I think we could consider making a term for something like 'response to quorum sensing molecule'. 

-  31 annotations from Mony et al. 2014 - This paper is about identifying genes of trypanosomes involved sensing their own population while in a host (which is necessary since you can't grow those life stages outside of a host); it does not describe any interaction of the trypanosomes with the host.

----------

**References:**
Mony et al. 2014. Genome-wide dissection of the quorum sensing signalling pathway in Trypanosoma brucei. Nature. 2505(7485):681-685. PMID:24336212 (31 IMP annotations)

Fujiya et al. 2007. The Bacillus subtilis quorum-sensing molecule CSF contributes to intestinal homeostasis via OCTN2, a host cell membrane transporter. Cell Host Microbe. 1(4):299-308.  PMID:18005709 (1 IMP annotation, 1 IDA annotation)

Bassler BL, Losick R. 2006. Bacterially speaking. Cell. 125(2):237-46. Review. PMID:16630813.

Camilli A, Bassler BL. 2006. Bacterial small-molecule signaling pathways. Science. 311(5764):1113-6. Review. PMID:16497924

_Originally posted by @krchristie in https://github.com/geneontology/go-ontology/issues/17730#issuecomment-524072591_

**Comment by krchristie** [comment]
Quorum sensing that just happens to occur while microbes are living in a host but only using the quorum sensing system to detect themselves, and possibly other microbial species, but not the host, does not seem like it should be annotated to `quorum sensing involved in interaction with host`. For many microbes, both parasitic and symbiotic, living in a host is a normal situation that is just the location where quorum sensing occurs, without quorum sensing being involved in any interaction with the host. 

In order to annotate to the term `quorum sensing involved in interaction with host`, it seems that **_direct interaction of the quorum sensing system with the host_** should be required, not just living in a host being the normal environment.

This quote from Bassler BL, Losick R. 2006. Bacterially speaking. (PMID:16630813) seems like quorum sensing that involves an interaction of the quorum sensing system with the host:
> (D) Conversations across kingdoms. Enterococcus faecalis produces a cytolysin composed of two subunits. The small cytolysin subunit acts as an autoinducer that monitors the environment for other E. faecalis cells. The large cytolysin subunit monitors the vicinity for eukaryotic host cells. Left: If no host cells are present, the two subunits form an inactive complex, and production of the subunits is held at a low basal level. Right: If host cells are present, the large subunit binds to the surface of the host cells, leaving the small subunit free to induce high-level production of both subunits, which together bind to target cells and cause them to lyse.

**Comment by krchristie** [comment]
I am leaning towards proposing obsoletion for this term
- None of the annotations to this term are for correct
-- The majority of annotations made to this term should have been made directly to 'quorum sensing' and had nothing to do with any interaction with the host; they just happened to occur in a host because that's where trypanosomes live
-- The other two annotations are for a host response to a quorum signaling molecule, but not in a way that affects quorum sensing by the microbe, nor was quorum sensing in the microbe addressed in any way in the paper
- This term isn't really even conceived correctly. 
-- I think that the term we would want to have as an is_a child of "quorum sensing" should be called "host interaction involved in quorum sensing" to indicate that some interaction with the host feeds into the quorum sensing system, such the Enterococcus faecalis example from Bassler & Losick 2006 that I included in the previous comment.
-- If we want a term for how the host reacts to the microbial quorum sensing system, that term should be a "response to other organism" term, or actually terms, since quorum sensing happens in both bacteria and fungi.

@ValWood - tagging you since you submitted the ticket that originated this issue
@pgaudet - wanted to check with you on obsoleting for these reasons
@RLovering - tagging you since I have challenged some annotations from your group to this term

**Comment by pgaudet** [comment]
I agree with obsoleting this term. 
Based on your second comment above, I also wonder if 'GO:0052097 interspecies quorum sensing' is correct ? There are no annotations. 

(There seem to be many related tickets - I am not sure on which to comment) 


We should also get @mgiglio99 's input.

Thanks, Pascale

**Comment by mgiglio99** [comment]
Karen, I think the example you found above (Bassler and Losick) supports some version of this term. I like your suggestion of a new term - I would reword it slightly to this:
"host interaction involving quorum sensing"
What do you think?

**Comment by krchristie** [comment]
@mgiglio99 - personally, I like "host interaction **involved in** quorum sensing" better than "... involving ..." since it seems to me to convey that the host detection is one of the signals that feeds into the quorum sensing pathway. 

Having used the word detection, I think I like this name even better: 
"host **detection** involved in quorum sensing"

But please explain your thoughts on why "involving" is better. I may be missing something.

**Comment by mgiglio99** [comment]
Hi @krchristie  
Using 'involved in quorum sensing' to me sounds like whatever is happening with the host is part of the quorum sensing process. While using 'involving' makes it a bit more broad such that quorum sensing is part of the interaction process but not necessarily the only part.

Regarding detection vs. interaction,  I worry that saying detection restricts things too much and I'd rather leave it open for more than just detection.

**Comment by krchristie** [comment]
Hi @mgiglio99 

I'm happy to leave the proposed new term for `host interaction involved in quorum sensing` more broad that just detection of the host. However, I'm concerned that `host interaction involving quorum sensing` has the potential to be overly broad and share the same problems that we've already seen for the existing term `quorum sensing involved in interaction with host` that we've agreed to obsolete. 

For the example I brought up from the Bassler and Losick paper, I am definitely trying to go for something that clearly indicates that the interaction of the host feeds into the quorum sensing system. So for a term to represent this type of situation, I definitely do prefer `host interaction involved in quorum sensing`.

Then, if the host detects and responds to the quorum sensing system, e.g. detection of a quorum signaling molecule produces a response in the host as described in Fujiya et al. 2007, I think the host protein should either be annotated to something like the existing term `response to bacterium`, OR if we want to highlight the fact that the host is responding to the quorum sensing system of its microbiome, then I think we should consider making a term for something like `response to quorum sensing molecule`.

**Comment by mgiglio99** [comment]
Sorry for the delay in responding to this.
I agree with your concern about 'involving' being too broad. 
So, let's go with 'host interaction involved in quorum sensing'

**Comment by krchristie** [comment]
Summary of actions for this ticket:
- [ ] propose obsoletion of `GO:0052106 - quorum sensing involved in interaction with host`

> There are 33 experimental annotations to this term by: 
> -- BHF-UCL (1 IDA, 1 IMP)
> -- GeneDB (31 IMP)
> 
> There are 0 InterPro2GO mappings to this term.
>  
> This term is NOT included in the any subsets.
> 
> This existing term is suggested for transferring annotations:
> -- GO:0009372 – quorum sensing


- [ ] create new term for `host interaction involved in quorum sensing`

**Comment by krchristie** [comment]
@mgiglio99 - We agreed on the name of the term, but didn't discuss a definition. This is what I've put in, but let me know if you have any suggestions. 

name: host interaction involved in quorum sensing
def: A quorum sensing process that is modulated by some interaction with a host cell or organism. [GOC:krc, GOC:mlg, PMID:16630813, PMID:11780122]

**Comment by mgiglio99** [comment]
Your def sounds fine.
Thanks
